.travis/install.sh

```diff
@@ -36,6 +36,9 @@ if [[ "$DARWIN" = true ]]; then
             curl -O https://bootstrap.pypa.io/get-pip.py
             sudo python get-pip.py
             ;;
+        bundle)
```

Comment:
I've addressed this below as well - but for clarity - the application builds fine using the system python.